source/sdk/Memory.ooc

```diff
@@ -2,17 +2,8 @@ include string // Memory routines in C are actually in string.h
 version(windows) { include malloc }
 else { include alloca }
 
-// GC_MALLOC zeroes the memory, so in the non-gc version, we prefer to use calloc
-// to the expense of some performance. Don't use malloc instead - existing ooc code
-// is written assuming that gc_malloc zeroes the memory.
-gc_malloc: func (size: SizeT) -> Pointer { gc_calloc(1, size) }
```

Comment:
Gotcha. I've never tried to do what you're suggesting, but I don't see how it would be different with `_malloc` as extern and `malloc` instead of `malloc` as extern and `gc_malloc`.